binaries/llm-cli/src/cli_args.rs

```diff
@@ -374,8 +382,16 @@ impl ModelLoad {
         let now = std::time::Instant::now();
         let mut prev_load_time = now;
 
+        let mut vocabulary_source = VocabularySource::ModelEmbedded;
+        if let Some(path) = &self.vocabulary_path {
+            vocabulary_source = VocabularySource::TokenizerFile(path.clone());
+        } else if let Some(repo) = &self.vocabulary_repo {
+            vocabulary_source = VocabularySource::TokenizerHfPretrained(repo.clone());
+        }
+
```

Comment:
Yeah I think it would be good if it were an error state if both of these are present...that means the user is confused.